src/world/world.c

```diff
@@ -0,0 +1,663 @@
+    { MAP_UNSPLIT(kzn_11, 0x80242680), .songVariation = 1, .flags = 2 },
+    { MAP_UNSPLIT(kzn_17, 0x802430C0), .songVariation = 1, .flags = 2 },
+    { MAP_UNSPLIT(kzn_18, 0x802403F0), .songVariation = 1, .flags = 2 },
+    { MAP_UNSPLIT(kzn_19, 0x80241D70), .songVariation = 1, .flags = 2 },
+    { MAP_UNSPLIT(kzn_20, 0x80240AE0), .songVariation = 1, .flags = 2 },
+    { MAP_UNSPLIT(kzn_22, 0x802403A0), .songVariation = 1, .flags = 2 },
+    { MAP_UNSPLIT(kzn_23, 0x80240630), .bgName = "yos_bg", .songVariation = 1, .flags = 2 },
+};
+
+/// Flower Fields
+static Map flo_maps[] = {
+    { MAP_UNSPLIT(flo_00, 0x80241490), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_03, 0x80240660), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_07, 0x80240760), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_08, 0x80241F00), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_09, 0x80241840), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_10, 0x802414A0), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_11, 0x802403A0), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_12, 0x80240710), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_13, 0x80243690), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_14, 0x802423B0), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_15, 0x80240020), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_16, 0x80242990), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_17, 0x80243240), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_18, 0x802407F0), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_19, 0x80240980), .bgName = "sra_bg" },
+    { MAP_UNSPLIT(flo_21, 0x80240D00), .bgName = "sra_bg" },
+    { MAP_UNSPLIT(flo_22, 0x802402A0), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_23, 0x80240CF0), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_24, 0x802405C0), .bgName = "fla_bg" },
+    { MAP_UNSPLIT(flo_25, 0x802422F0), .bgName = "fla_bg" },
+};
+
+/// Shiver Region
+static Map sam_maps[] = {
+    { MAP_UNSPLIT(sam_01, 0x80241F70), .bgName = "yki_bg" },
+    { MAP_UNSPLIT(sam_02, 0x80241C00), .bgName = "yki_bg" },
+    { MAP_UNSPLIT(sam_03, 0x80240940), .bgName = "yki_bg" },
+    { MAP_UNSPLIT(sam_04, 0x80240740), .bgName = "yki_bg" },
+    { MAP_UNSPLIT(sam_05, 0x80241D60), .bgName = "sam_bg" },
+    { MAP_UNSPLIT(sam_06, 0x80242630), .bgName = "sam_bg" },
+    { MAP_UNSPLIT(sam_07, 0x80242020), .bgName = "yki_bg", .songVariation = 1, .flags = 1 },
+    { MAP_UNSPLIT(sam_08, 0x802417D0), .bgName = "yki_bg", .songVariation = 1, .flags = 1 },
+    { MAP_UNSPLIT(sam_09, 0x80240420), .bgName = "yki_bg", .songVariation = 1, .flags = 1 },
+    { MAP_UNSPLIT(sam_10, 0x80240CE0), .bgName = "yki_bg", .songVariation = 1, .flags = 1 },
+    { MAP_UNSPLIT(sam_11, 0x802421A0), .bgName = "yki_bg" },
+    { MAP_UNSPLIT(sam_12, 0x802403B0), .bgName = "yki_bg", .songVariation = 1, .flags = 1 },
+};
+
+/// Crystal Palace
+static Map pra_maps[] = {
+    { MAP_UNSPLIT(pra_01, 0x80241400), .bgName = "yki_bg", .flags = 1, .init = (MapInitialiser*)0x80240000 },
```

Comment:
I'm just seeing this (`MapInitialiser`) now, not sure if it's new, but we should probably have a conversation about whether we want to use British English or American English for the code itself. I don't think it's as important that we be consistent in the comments, but we should be in the code.